core/src/main/java/org/infinispan/transaction/impl/TransactionTable.java

```diff
@@ -277,37 +277,41 @@ public int getMinTopologyId() {
    }
 
    public void cleanupLeaverTransactions(List<Address> members) {
-      // Can happen if the cache is non-transactional
-      if (remoteTransactions == null)
-         return;
+      try {
+         // Can happen if the cache is non-transactional
+         if (remoteTransactions == null)
```

Comment:
I added the try block to make sure we don't throw any exception here that would interrupt the topology update in `StateConsumerImpl`.